sdk/go/common/workspace/creds.go

```diff
@@ -120,14 +121,60 @@ type Account struct {
 	Insecure bool `json:"insecure,omitempty"`
 	// Information about the token used to authenticate.
 	TokenInformation *TokenInformation `json:"tokenInformation,omitempty"`
+	// Additional context used during authentication, like token exchange details.
+	AuthContext *AuthContext `json:"authContext,omitempty"`
 }
 
 // Information about the token that was used to authenticate the current user. One (or none) of Team or Organization
 // will be set, but not both.
 type TokenInformation struct {
-	Name         string `json:"name"`                   // The name of the token.
-	Organization string `json:"organization,omitempty"` // If this was an organization token, the organization it was for.
-	Team         string `json:"team,omitempty"`         // If this was a team token, the team it was for.
+	Name         string     `json:"name"`                   // The name of the token.
+	Organization string     `json:"organization,omitempty"` //nolint:lll // If this was an organization token, the organization it was for.
+	Team         string     `json:"team,omitempty"`         // If this was a team token, the team it was for.
+	ExpiresAt    *time.Time `json:"expiresAt,omitempty"`    // The time when this token expires.
+}
+
+type AuthContext struct {
+	GrantType    string        `json:"grantType,omitempty"`
+	Organization string        `json:"organization,omitempty"`
+	Scope        string        `json:"scope,omitempty"`
+	Token        string        `json:"token,omitempty"`
+	TokenExpired bool          `json:"tokenExpired,omitempty"`
+	Expiration   time.Duration `json:"expiration,omitempty"`
```

Comment:
I think this will JSON serialize in milliseconds, and time.Time as "YYYY-MM-DDTHH:MM:SSZ" that's probably fine, but just pointing out it won't be seconds/epoch time like you originally intended.